README.md

```diff
@@ -0,0 +1,125 @@
+# TAMS Master Project 2022/2023
+
+This repository contains the codebase for a [Marimba](https://en.wikipedia.org/wiki/Marimba) playing robot developed during the TAMS Master Project 2022/2023. 
+It is currently still "work in progress".
+
+## Setup 
+
+The robot uses Ubuntu 20.04 and ROS noetic. 
+So make sure that it is installed on your system.
+Visit the [ROS Wiki](http://wiki.ros.org/noetic/Installation) for further details.
+
+Remember to source the ROS noetic installation in EACH terminal you use or add it once to your `.bashrc` file like this:
```

Comment:
would it make sense to provide such file?

Reply:
The file is already present and specific to your system configuration. 

The lines below just append the command to the file and reload it.